plugins/pulp_rpm/plugins/importers/yum/sync.py

```diff
@@ -1070,7 +1071,10 @@ def _identify_wanted_versions(self, package_info_generator):
 
         for model in package_info_generator:
             if skip_srpm and isinstance(model, models.SRPM):
-                number_of_unique_packages += 1
+                unit_key = models.NEVRA._fromdict(model.unit_key)
```

Comment:
@daviddavis , maybe it's nitpicking... I have one minor concern.
IIRC, when NEVRA class was introduced (we worked with Sean on errata stuff), it turned out to be quite expensive to use it, so we tried to reduce the number of these @_fromdict@ calls.
Having said that, I'm not sure it is worth rewriting this part but I'm not sure how many SRPMs are actually skipped, tons or just some. 
Alternatively, can you use just SRPM checksum to track them?

Reply:
After IRC discussion, it turned out that packages can have the same nevra and different checksums. It's a problem and a bug of the upstream repo but we are trying to handle it in pulp.